src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1767,6 +1768,7 @@ void goto_instrument_parse_optionst::instrument_goto_program()
   {
     do_indirect_call_and_rtti_removal();
     do_remove_returns();
+    rewrite_rw_ok(goto_model);
```

Comment:
I see only the `full-slice` option is rewriting the `rw` expressions.

Is there some specific reason the other slices don't want to display the same functionality? Apologies, but I couldn't tell the intent here just by reading the code.

Reply:
Other forms of slicing don't use expression-level granularity. I have updated the commit message to include this information.